localization/yabloc/yabloc_common/src/extract_line_segments.cpp

```diff
@@ -21,12 +21,11 @@ pcl::PointCloud<pcl::PointNormal> extract_near_line_segments(
   const float max_range)
 {
   constexpr double sqrt_two = 1.41421356237309504880;
-  const Eigen::Vector3f pose_vector = transform.translation();
+  const Eigen::Vector3f & pose_vector = transform.translation();
 
   // All line segments contained in a square with max_range on one side must be taken out,
   // so pick up those that are closer than the **diagonals** of the square.
-  auto check_intersection = [sqrt_two, max_range,
-                             pose_vector](const pcl::PointNormal & pn) -> bool {
+  auto check_intersection = [max_range, pose_vector](const pcl::PointNormal & pn) -> bool {
```

Comment:
@veqcc Hi. The PR auther is fixing the all issues pointed out by clang-tidy, but here it's suggesting to revert the changes you made last time. https://github.com/autowarefoundation/autoware.universe/pull/7208
What do you think we should do?

Here is clang-tidy' feedback 
```
/home/masakibaba/autoware/src/universe/autoware.universe/localization/yabloc/yabloc_common/src/extract_line_segments.cpp:28:30: error: lambda capture 'sqrt_two' is not required to be captured for this use [clang-diagnostic-unused-lambda-capture]
  auto check_intersection = [sqrt_two, max_range,
```

Reply:
@KYabuuchi 
Thank you for informing me.

Unfortunately, I'm not sure how to solve it...
How about changing the `sqrt_two` from constexpr to const variable?
I have read the article: https://stackoverflow.com/questions/43467095/why-is-a-const-variable-sometimes-not-required-to-be-captured-in-a-lambda

Reply:
Thank you for the advice! 
In my local environment when I changed constexpr to const and used explicit capturing, the clang-tidy error was resolved.  I think this solution is fine :+1:

```cpp
const double sqrt_two=1.414;
auto check_intersection = [sqrt_two ...
```
@a-maumau  Could you fix it that way?

Reply:
Thank you for the suggestion!
I changed it at da1e6fa .